r11701/default_common_inlists/binary/src/run_binary_extras.f

```diff
@@ -1145,6 +1146,18 @@ integer function extras_binary_startup(binary_id,restart,ierr)
             end if
          end if
          extras_binary_startup = keep_going
+
+         if (b% doing_first_model_of_run .and. b% terminate_if_initial_overflow &
+                  .and. (.not. b% ignore_rlof_flag .or. b% model_twins_flag)) then
+              rl_gap_1 = (b% s1% photosphere_r - b% rl(1)/Rsun * (1 - b% eccentricity) )/(b% rl(1)/Rsun)
```

Comment:
Is there a reason, why you use `s1% photosphere_r` here instead of `r(1)`? If not, I'd suggest to use `r(1)`, which is in the same units as `rl(1)`, hene no need for additional unit conversions.
Minor: Why has this line a different intend then the following one?

Reply:
Good question. Using the s% photosphere and not the s% r(1) is crucial for catching the initial_RLO, as what I understood is that s% (1) has not been updated to the new value after relaxation.